doc/whats_new.rst

```diff
@@ -1,3 +1,21 @@
+0.4.3
+=====
+
+Changes
+-------
+
+    - :func:`nilearn.datasets.fetch_surf_fsaverage5` is deprecated and will be
+      removed in a future release. Use `nilearn.fetch_surf_fsaverage`, with the
```

Comment:
changing `nilearn.fetch_surf_fsaverage` to :func:`nilearn.datasets.fetch_surf_fsaverage` could fix CircleCI failure. I think.

Reply:
cool, thanks!